libcontainer/configs/config_test.go

```diff
@@ -145,12 +145,15 @@ func TestFuncHookRun(t *testing.T) {
 
 	fHook := configs.NewFunctionHook(func(s *specs.State) error {
 		if !reflect.DeepEqual(state, s) {
-			t.Errorf("Expected state %+v to equal %+v", state, s)
+			return fmt.Errorf("expected state %+v to equal %+v", state, s)
 		}
 		return nil
 	})
 
-	fHook.Run(state)
+	err := fHook.Run(state)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Not important, but the code used to keep checking other cases even after one of them failed, and now it's not.

Fine either way for me, just noticing.

Reply:
Hmm... but there's only a single check in this test, and if that would fail it would already fail the test.

I _think_ it's correct with this change, so I'll keep it, but feel free to comment if I overlooked something 😅